google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/ParquetOptions.java

```diff
@@ -56,6 +64,19 @@ public Builder setEnumAsString(Boolean enumAsString) {
       return this;
     }
 
+    /**
+     * [Optional] Indicates how to represent a Parquet map if present.
```

Comment:
Can we document a URL for more info?

Reply:
Changed comment to include document URL instead.